### ISSUE TITLE
gutcheck versions of underlying code bases and update as appropriate

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1892]
* axe core 
* puppeteer 
* anything else?

**Comment by ethangardner** [comment]
@gbinal - The currently installed versions will allow us to complete the accessibility work. We should be good to go without blockers. However, one of the requested scans, [`css-orientation-lock`](https://dequeuniversity.com/rules/axe/4.12/css-orientation-lock) is still in the experimental stage in `axe-core` . I recommend we hold off on adding that one because the results could change until they make it stable. The rest of the requested ones should be fine.

**Comment by gbinal** [comment]
Rock - thanks for checking all this!  Good work.
